### PR TITLE
fix(specs): correct `searchSynonyms` parameters

### PR DESCRIPTION
**Pull request by morganleroi**
## 🧭 What and Why

🎟 JIRA Ticket: -

### Changes included:

The parameters of the `searchSynonyms` method are in fact body parameters and not query parameters, this PR also fixes the browse helpers

### CODE REVIEW

**Comment by morganleroi** [comment]
@millotp @shortcuts do you an idea why this change broke the CTS generation ?

File: specs/search/paths/synonyms/searchSynonyms.yml [hunk 1]
```diff
@@ -24,6 +21,12 @@ post:
           properties:
             query:
               $ref: '../../../common/schemas/SearchParams.yml#/query'
+            type:
+              $ref: './common/parameters.yml#/SynonymType'
```
Comment (shortcuts) [inline specs/search/paths/synonyms/searchSynonyms.yml:25]: I guess `type` is reserved in many languages and fails?

Comment (morganleroi) [reply specs/search/paths/synonyms/searchSynonyms.yml:25]: But it's only the CTS that fails to generate. `Error in searchSynonyms with all parameters: Parameter type not found in the root parameter`

I'll try to dig into the generator.

Comment (shortcuts) [reply specs/search/paths/synonyms/searchSynonyms.yml:25]: yup that's weird but if you try to gen the python client it doesn't work 🤔

**Review by shortcuts — APPROVED** [review]
great catch!! gg

**Pull request merged**